vault/resource_gcp_secret_impersonated_account.go

```diff
@@ -68,6 +68,12 @@ func gcpSecretImpersonatedAccountResource() *schema.Resource {
 				Computed:    true,
 				Description: "Project of the GCP Service Account managed by this impersonated account",
 			},
+			consts.FieldTTL: {
```

Comment:
This should probably be `Computed: True` since Vault sets a default

Reply:
This has now been added.